libaleth-interpreter/VM.cpp

```diff
@@ -1416,7 +1420,10 @@ void VM::interpretCases()
         CASE(INVALID)
         DEFAULT
         {
-            throwBadInstruction();
+            if (m_OP == Instruction::INVALID)
```

Comment:
Shouln't it be just

```
CASE(INVALID)
  throwInvalidInstruction();
DEFAULT
  throwUndefinedInstruction();
```

Reply:
No, this doesn't work on gcc/clang, because the jump table is populated with `INVALID` opcode for all unknown ones https://github.com/ethereum/aleth/blob/089e7ddaa741825fdf62974ad66b077c715b8f25/libaleth-interpreter/VMConfig.h#L154

Reply:
Can't we change that by having `&&UNDEFINED` everywhere and `&&INVALID` in https://github.com/ethereum/aleth/blob/089e7ddaa741825fdf62974ad66b077c715b8f25/libaleth-interpreter/VMConfig.h#L409?

Reply:
There's no such thing as `UNDEFINED`, and this enum is 8 bit-wide https://github.com/ethereum/aleth/blob/2b11ce5c5aaf00f54ce9a301f8c103148d00abdc/libevm/Instruction.h#L28

So we can't really add `UNDEFINED` unless we change the width